sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestRecord.java

```diff
@@ -275,7 +290,7 @@ public String toString() {
     // region Types
 
     public enum Stage {
-        QUEUED, PIPELINED, SENT, RECEIVED, COMPLETED
+        QUEUED, ACQUIRECHANNELSTARTED, PIPELINED, SENT, RECEIVED, COMPLETED
```

Comment:
Rename to -> `CHANNEL_ACQUISITION_STARTED` ?

Also, we don't have `Channel Acquisition Ended` state, so should we rename this to `CHANNEL_ACQUIRED` ?

Reply:
yea, I am open for name change. 

Hmm, but do not think should change to CHANNEL_ACQUIRED, because I changed the stage when we started to acquire channel.

I did not add channel_acquision_end, because pipelined stage should be a good indicate channel has acquired. But open to change~

Reply:
updated to CHANNEL_ACQUISION_STARTED